sphinx_search/static/js/rtd_sphinx_search.js

```diff
@@ -1,6 +1,9 @@
 const MAX_SUGGESTIONS = 50;
 const MAX_SECTION_RESULTS = 3;
 const MAX_SUBSTRING_LIMIT = 100;
+const ANIMATION_TIME = 200;
+const FETCH_RESULTS_DELAY = 250;
+const CLEAR_RESULTS_DELAY = 300;
```

Comment:
Any reason this isn't the same as `FETCH_RESULTS_DELAY`?

Reply:
They both can collide.

https://github.com/readthedocs/readthedocs-sphinx-search/blob/08cc10513093b9acd2ba1f4e8e3d27f1dfb66fc0/sphinx_search/static/js/rtd_sphinx_search.js#L659-L665

So, we need to wait a little more to make sure the results are cleared.